### PR TITLE
Happy Birthday Henry

### PR DESCRIPTION
**Pull request by kj800x**
|Before|After|
|-|-|
|image|image|

### CODE REVIEW

**Comment by kj800x** [comment]
Hows this look? @Jmeferlater @jakecard20 @sambart19

**Review by Jmeferlater — APPROVED** [review]
LGTM

File: script.js [hunk 1]
```diff
@@ -34,20 +34,27 @@ const QUOTES = [
...
   const queryParams = new URL(document.location).searchParams;
 
   if (queryParams.get('quote') !== null) {
-    return parseInt(queryParams.get('quote'));
+    return QUOTES[parseInt(queryParams.get('quote'))];
   }
 
   const today = new Date();
+
+  if (today.getDate() === 20 && today.getMonth() === 2) {
```
Comment (Jmeferlater) [inline script.js:46]: unrelated but 0 index'ing things that are literally already numbered is stupid i hate it

**Comment by jakecard20** [comment]
I'm glad your able to bring such a beautiful thing into this world. Looks good

**Review by sambart19 — APPROVED** [review]
🚢

File: script.js (hunk 1 above)
Comment (sambart19) [reply script.js:46]: The best part is date isn't 0 indexed but month is lol

**Comment by kj800x** [comment]
I'll call that critical mass

File: script.js (hunk 1 above)
Comment (kj800x) [reply script.js:46]: Literally the worst I hate it so much

**Pull request merged**